src/runtime/internal/animations.ts

```diff
@@ -1,4 +1,5 @@
-import { identity as linear, noop, now } from './utils';
+import { identity as linear, noop } from './utils';
+import { now } from "./environment";
```

Comment:
Absolutely a nit, but can you use single quotes here and in the other places where you've added imports?

Reply:
Definitely! That must've happened because of vscode's auto-fix-imports thing. I never use double quotes myself. And one more reason we need to get all of eslint's rules decided so we can add it to test.